portfolio/src/main/webapp/script.js

```diff
@@ -60,6 +51,61 @@ async function getCommentFromFetch() {
         var para = document.createElement("p");
         var node = document.createTextNode(commentString);
         para.appendChild(node);
+        var button = document.createElement("button");
```

Comment:
Because these buttons belong to a form, their default behavior is to submit the form. So, unless the click handler stops the default event propagation (or returns false?), the form will be submitted, in addition the `likeComments` function being called to do the `fetch`. More about button types here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type

I don't think this was intentional; probably you want the Like button to just apply the Like in the database. If so, it would be good to make it not reload the page, as well.

Reply:
I didn't know that the default type of a button is "submit" and when using in the form it will automatically submit the form, which will lead to the page reloading. I change the type of the button to "button" and now it will not submit the form and reload the page.